solutionbox/structured_data/datalab_solutions/structured_data/trainer/task.py

```diff
@@ -262,40 +140,48 @@ def get_experiment(output_dir):
     # Get the model to train.
     estimator = util.get_estimator(output_dir, train_config, args)
 
-    input_placeholder_for_prediction = get_placeholder_input_fn(
-        train_config, args.preprocess_output_dir, args.model_type)
-
     # Save a copy of the scehma and input to the model folder.
     schema_file = os.path.join(args.preprocess_output_dir, util.SCHEMA_FILE)
 
     # Make list of files to save with the trained model.
-    additional_assets = [args.transforms_file, schema_file]
+    additional_assets = {'transforms.json': args.transforms_file, 
+                         util.SCHEMA_FILE: schema_file}
     if util.is_classification_model(args.model_type):
       target_name = train_config['target_column']
+      vocab_file_name = util.CATEGORICAL_ANALYSIS % target_name
       vocab_file_path = os.path.join(
-          args.preprocess_output_dir, util.CATEGORICAL_ANALYSIS % target_name)
+          args.preprocess_output_dir, vocab_file_name)
       assert file_io.file_exists(vocab_file_path)
-      additional_assets.append(vocab_file_path)
-
-    # Save the finished model to output_dir/model
-    export_monitor = util.ExportLastModelMonitor(
-        output_dir=output_dir,
-        final_model_location='model',  # Relative to the output_dir.
-        additional_assets=additional_assets,
-        input_fn=input_placeholder_for_prediction,
-        input_feature_key=FEATURES_EXAMPLE_DICT_KEY,
-        signature_fn=get_export_signature_fn(train_config, args))
-
-    #final_export_strategy = make_export_strategy(
-    #  serving_input_fn=input_placeholder_for_prediction, 
-    #  exports_to_keep=5)
+      additional_assets[vocab_file_name] = vocab_file_path
+
+    export_strategy_csv = util.make_export_strategy(
+        train_config=train_config, 
+        args=args, 
+        keep_target=True,
+        assets_extra=additional_assets)
+    export_strategy_json = util.make_export_strategy(
+        train_config=train_config, 
+        args=args, 
+        keep_target=False,
```

Comment:
This is an old name. Renamed  to export_strategy_target, export_strategy_notarget